linux-installer.sh

```diff
@@ -239,7 +246,12 @@ fi
 PRETTY_WORKER_NAME=$WORKER_ID
 if [[ -z ${WORKER_ID+x} ]]; then
   echo "WORKER_ID was not defined, using random numeric string ..."
-  PRETTY_WORKER_NAME="<random string>"
+  if is_darwin; then
+     random=$(cat /dev/urandom | env LC_CTYPE=C tr -dc '0-9' | fold -w 16 | head -n 1)
+  else
+     random=$(cat /dev/urandom | tr -dc '0-9' | fold -w 16 | head -n 1)
+  fi
+  PRETTY_WORKER_NAME=$random
```

Comment:
Lol that's not what I actually wanted, but interesting.

`PRETTY_WORKER_NAME` should be used only for display; not for actually assigning a value. It was to prevent a blank string from displaying in the initial output; ie:

```
Wallet: NQ33 ...
Worker ID:
```

With this `PRETTY_WORKER_NAME` we can just give it `<random string>` I guess. When the pool server picks up a worker without a name set (ie. without `--extra-data`) it will automatically assign an integer-based random ID to it.

Reply:
Ohhh whoops, I'll change it back